### DISCUSSION TITLE
Are flux Kustomization objects suppose to be created separately or part of the flux source git repo?

### DISCUSSION THREAD

**Discussion by tachang** [fluxcd/flux2#1212] [Q&A] [answered]
Are flux Kustomization objects suppose to be created separately or part of the flux source git repo?

The object I am talking about is part of `kustomize.toolkit.fluxcd.io/v1beta1`.

It sounds like they are suppose to be created outside of the git repo but that doesn't make much sense does it?

**Comment by Frederik-Baetens** [comment, marked as answer]
It's all supposed to be in the git repo. If you were to use flux bootstrap, as recommended in the [Get started guide](https://toolkit.fluxcd.io/get-started/) these files will automatically be created in your git repository for you.
